src/meta/processors/admin/Balancer.cpp

```diff
@@ -471,9 +471,12 @@ cpp2::ErrorCode Balancer::leaderBalance() {
                 }
             }
         }).wait();
-        LOG(ERROR) << failed << " partiton failed to transfer leader";
         inLeaderBalance_ = false;
-        return cpp2::ErrorCode::E_BALANCER_FAILURE;
+        if (failed > 0) {
```

Comment:
Good catch！

Reply:
Hold on, @laura-ding, this place is intended to do that, we don't expect all transfer leader is successful, no need to return failure.